wcivf/apps/hustings/templates/hustings/includes/_ld_husting.html

```diff
@@ -1,21 +1,31 @@
 {% load static %}
 <script type="application/ld+json">
-{
-    "@context": "http://schema.org/",
-    "@type": "Event",
-    "url" : "{{ CANONICAL_URL }}{{ husting.post_election.get_absolute_url }}#hustings",
-    "image": "{{ CANONICAL_URL }}{% static "dc_theme/images/logo.png" %}",
-    "startDate": "{{ husting.starts | date:'Y-m-d H:i' }}",
-{% if husting.ends %}
-    "endDate": "{{ husting.ends | date:'Y-m-d H:i' }}",
-{% endif %}
-    "name": "{{ husting.title}}",
-    "description": "Husting event for {{ husting.post_election.friendly_name}}",
-{% if husting.location and husting.postcode %}
-    "location" : {
-        "@type" : "Place",
-        "address": "{{ husting.location }}, {{ husting.postcode }}, UK"
+    {
+        "@context": "http://schema.org/",
+        "@type": "Event",
+        "url": "{{ CANONICAL_URL }}{{ husting.post_election.get_absolute_url }}#hustings",
+        "image": "{{ CANONICAL_URL }}{% static "
+        dc_theme / images / logo.png " %}",
+        "startDate": "{{ husting.starts | date:'Y-m-d H:i' }}",
+        {
```

Comment:
Same problem with the Django templating language in this file as mentioned above - needs to be reverted